src/app/beneficiaries/beneficiaries-routing.module.ts

```diff
@@ -13,13 +15,25 @@ const routes: Routes = [
       component: BeneficiariesComponent,
       data: {title: extract('Beneficiaries')},
       resolve: { beneficiaries: BeneficiariesResolver}},
+    {
+      path: 'beneficiaries/create',
+      component: BeneficiariesFormComponent,
+      data: {title: extract('Add Beneficiary')}, // think of a way to use this
```

Comment:
@ksraj123 use of this object is intended for the iteration to achieve internationalization (multi language support).